en/config_mc/pid_tuning_guide_multicopter.md

```diff
@@ -53,6 +16,11 @@ Here are some general points to follow when tuning:
 - Land before changing a parameter.
```

Comment:
For the basic tuning we don't mention this. Should all the tuning be done on the ground there too?

Reply:
Hmm. I think perhaps I'll leave this for now.